template-tags.php

```diff
@@ -417,19 +417,41 @@ function coauthors_ids( $between = null, $betweenLast = null, $before = null, $a
 	), null, $echo );
 }
 
-function get_the_coauthor_meta( $field ) {
+/**
+ * Outputs the co-authors Meta Data
+ *
+ * @param string $field Required The user field to retrieve.[login, email, nicename, display_name, url, type]
+ */
+function get_the_coauthor_meta( $field ) { 
```

Comment:
Since `the_coauthor_meta` passes in a possible second parameter to `get_the_coauthor_meta` called `$user_id`, perhaps we should take that into account.